ceph-salt-formula/salt/ceph-salt/apply/tuned-latency.sls

```diff
@@ -0,0 +1,40 @@
+
+{% import 'macros.yml' as macros %}
+
+{% if 'latency' in grains['ceph-salt']['roles'] %}
+
+{{ macros.begin_stage('Configuring tuned latency') }}
+
+install tuned:
+  pkg.installed:
+    - pkgs:
+      - tuned
+    - failhard: true
+
+/etc/tuned/ceph-latency/tuned.conf:
+  file.managed:
+    - source: salt://ceph-salt/apply/files/latency.conf.j2
+    - template: jinja
+    - makedirs: True
+    - user: root
+    - group: root
+    - mode: 644
+    - failhard: true
+
+start tuned for latency profile:
+  service.running:
+    - name: tuned
+    - enable: True
+    - failhard: true
+
+apply latency profile:
+  cmd.run:
+    - name: 'tuned-adm profile ceph-latency'
+    - failhard: true
+
+{{ macros.begin_stage('Configuring tuned latency') }}
```

Comment:
This should be an end_stage: `s/begin_stage/end_stage/`

Also, for consistency with other stages, can we `s/Configuring/Configure/` (same applies to other stages that you added?